### PR TITLE
fix paths

### PR DESCRIPTION
**Pull request by TezRomacH**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -109,10 +109,8 @@ ENV/
 tmp/
 logs/
 
-*.ipynb
```
Comment (TezRomacH) [inline .gitignore]: this one is for to be able to create new examples

File: .gitignore [hunk 2]
```diff
@@ -109,10 +109,8 @@ ENV/
...
 *.csv
 *.tsv
-*.txt
```
Comment (TezRomacH) [inline .gitignore]: this one for updating `requirements.txt`

File: .gitignore (hunk 1 above)
Comment (Scitator) [reply .gitignore]: all current examples already added and tracked by git
for new ones, you can easily use `git add -f examples/new_ex.ipynb`

File: .gitignore (hunk 2 above)
Comment (Scitator) [reply .gitignore]: just like with .ipynb, all current requirements already tracked

**Pull request merged**